Xamarin.Essentials/Barometer/Barometer.uwp.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Windows.Devices.Sensors;
+using WinBarometer = Windows.Devices.Sensors.Barometer;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Barometer
+    {
+        static readonly Lazy<WinBarometer> sensor = new Lazy<WinBarometer>(() => WinBarometer.GetDefault());
```

Comment:
Same, we don't want to use lazy here. And this should be 

```
internal static WindowsBarometer DefaultBarometer => WindowsBarometer.GetDefailt();
```

Reply:
I just checked the uwp compass inplementation. We actually do not reset the actual sensor. We just use our own version of a lazy.

Reply:
Let me evaluate our lazy usage.